rgo/builders/copr.py

```diff
@@ -53,39 +53,42 @@ def __init__(
         for chroot in chroots:
             if chroot not in self.client.mock_chroot_proxy.get_list():
                 raise Exception("{!r} is not an active chroot".format(chroot))
-        self.chroots = chroots
+        self.cmdline_input_chroots = chroots
 
         self.enable_net = enable_net
 
         try:
             self.project = self.client.project_proxy.get(owner, name)
             LOGGER.info("Using existing COPR project: %s/%s", self.project.ownername, self.project.name)
 
-            # add the build chroots to the project's chroots and extend its expiration time
+            self.all_project_chroots = set(list(self.project.chroot_repos.keys()) + self.cmdline_input_chroots)
```

Comment:
This only sets `self.all_project_chroots` if the project exists (otherwise an exception is thrown above and you don't set it). You later rely on this being there, so I think it will fail if you call this on a non-existent project.